File: C/1149.c
#include<stdio.h>
int main()
{
    int A,N,sum=0;
    scanf("%d %d",&A,&N);
    while(N<=0)
    {
        scanf("%d",&N);
    }

    for(int i=0;i<N;i++)
    {
        sum+=(A+i);
    }
    printf("%d\n",sum);
    return 0;
}

File: Python/1149.py
l=list(map(int,input().split()))
a='I'
sum=0
for i in l:
    if(a=='I'):
        a=int(i)
    elif(i>0):
        n=int(i)
        break
for i in range(n):
    sum+=a
    a+=1
print(sum)